### ISSUE TITLE
How to get all user's media?

### ISSUE THREAD

**Issue by kheniparth** [charlieAndroidDev/Instagram4Android#57]
Hey I am new to this repo, could you please mention any request name or anything that can help me to fetch user's media.

Thank you :)

**Comment by mhosseinab** [comment]
`InstagramFeedResult feedResult = instagram.sendRequest(new InstagramUserFeedRequest(userId, maxId, minTimeStamp));`

**Comment by kukalajet** [comment]
@mhosseinab What's _maxId_?

**Comment by mhosseinab** [comment]
@kukalajet in the first request _maxId=null_ and you will get a new value in the result set.
The _maxId_ is the key parameter to handle pagination with instagram and works like a pointer to the query results.

**Comment by vadimTovstik** [comment]
@mhosseinab maxId specifies count results per page?

I try to get all followers, without maxId parameter I get only 200 followers, if I specify maxId parameter the program runs infinitely in this place.
`InstagramGetUserFollowersResult followersResult = 
mApiInstagram.sendRequest(new InstagramGetUserFollowersRequest(user.getPk(), "2"));`

**Comment by mhosseinab** [comment]
No, maxId is the end cursor to the page in which you have received the data. 
for example, you get 200 records and a maxId in the response, if you need to get the next 200, the maxId received in the reply should be used to get the next page. 

Please note that there is a huge amount of data there and you can only get data in pages. The amount of data on each page varies for each endpoint or even they change it day by day!
